graph/graph_windows.go

```diff
@@ -114,7 +119,29 @@ func (graph *Graph) storeImage(img *Image, layerData archive.ArchiveReader, root
 
 		// Store the layer. If layerData is not nil, unpack it into the new layer
 		if layerData != nil {
-			if img.Size, err = graph.driver.ApplyDiff(img.ID, img.Parent, layerData); err != nil {
+			// this is saving the tar-split metadata
```

Comment:
This code block is identical to the one in `graph_unix. Seems odd to be making that split at this level. Perhaps, it is too late...